compiler/crates/relay-transforms/src/client_edges.rs

```diff
@@ -363,7 +363,18 @@ impl<'program, 'sc> ClientEdgesTransform<'program, 'sc> {
             }
 
             match edge_to_type {
-                Type::Interface(_) => {
+                Type::Interface(interface_id) => {
+                    let interface = schema.interface(interface_id);
+                    let implementing_objects =
+                        interface.recursively_implementing_objects(Arc::as_ref(schema));
```

Comment:
Can we add a test case which covers the case where the interface is implemented by another interface, but there are still no concrete implementations? Should still error.

Reply:
added a test case called `relay-resolver-edge-to-interface-with-child-interface-and-no-implementors`!